doc/release/upcoming_changes/16102.improvement.rst

```diff
@@ -0,0 +1,4 @@
+``np.logaddexp2.identity`` changed to ``-inf``
+----------------------------------------------
+The ufunc `logaddexp2` now has an identity of ``-inf``, allowing it to
+be called on empty sequences.  This matches the identity of `logaddexp`.
```

Comment:
Thanks!
```suggestion
The ufunc `~numpy.logaddexp2` now has an identity of ``-inf``, allowing it to
be called on empty sequences.  This matches the identity of `~numpy.logaddexp`.
```
will make that a valid link, without it I do not think it works because this is defined in different corners of the documentation.

Reply:
Neat, thanks.  I'll trust you on this.